Firmware/mmu2.cpp

```diff
@@ -166,7 +167,14 @@ void MMU2::mmu_loop() {
     avoidRecursion = true;
 
     logicStepLastStatus = LogicStep(); // it looks like the mmu_loop doesn't need to be a blocking call
-    
+
+    if (is_mmu_error_monitor_active)
```

Comment:
:thinking: not sure if this is the right way of keeping the MMU error screen responsive ... but probably ok for now